e2e-tests/haproxy/run

```diff
@@ -51,6 +51,8 @@ main() {
 
 	desc 'delete active writer and checking all haproxy pods still point to the same writer'
 	kubectl_bin delete pod $cluster-pxc-0
+	sleep 3
+	kubectl_bin get pods
```

Comment:
Could we add a description with problems to test we want to change? Like TO DO or something? It might increase chances to fix it together with other task. What do you think?

Reply:
We can do that, although I would rather create Jira tickets and assign to whoever, this way it will not just be written somewhere and forgotten.